gemrb/plugins/GAMImporter/GAMImporter.cpp

```diff
@@ -755,16 +755,14 @@ int GAMImporter::PutPlaneLocations(DataStream *stream, Game *game) const
 //only in PST
 int GAMImporter::PutKillVars(DataStream *stream, const Game *game) const
 {
-	ieVariable tmpname;
 	Variables::iterator pos=NULL;
 	const char *name;
 	ieDword value;
 
 	for (unsigned int i=0;i<KillVarsCount;i++) {
 		//global variables are locals for game, that's why the local/global confusion
 		pos=game->kaputz->GetNextAssoc( pos, name, value);
-		strnspccpy(tmpname, name, 32, core->HasFeature(GF_NO_NEW_VARIABLES));
-		stream->WriteVariable(tmpname);
+		stream->WriteVariable(ieVariable(name));
```

Comment:
Looks like this doesn't preserve the current behaviour. Ignoring the space removal is likely fine, since I guess you checked that we do the same already when reading them, so the runtime vars don't have spaces.

But the uppercasing was deemed important for PST, so we should leave that intact until proven redundant.